scss/_patterns_footer.scss

```diff
@@ -42,7 +48,6 @@
     &__link {
       border-bottom: 0;
       color: $color-dark;
```

Comment:
@anthonydillon I think the footer should work on a white or grey background without setting colour